src/sentry/features/temporary.py

```diff
@@ -561,6 +561,8 @@ def register_temporary_features(manager: FeatureManager) -> None:
     manager.add("organizations:ingest-through-trusted-relays-only", OrganizationFeature, FeatureHandlerStrategy.FLAGPOLE, api_expose=True)
     # Enable new workflow_engine UI (see: alerts create issues)
     manager.add("organizations:workflow-engine-ui", OrganizationFeature, FeatureHandlerStrategy.FLAGPOLE, api_expose=True)
+    # Disable redirects from alert rules to the new workflow_engine UI
+    manager.add("organizations:workflow-engine-redirect-opt-out", OrganizationFeature, FeatureHandlerStrategy.FLAGPOLE, api_expose=True)
```

Comment:
rather than having this be `opt-out` and `True`, could we have this be `opt-in`? The negation of the boolean expression can add to a bit of confusion, so naming this to be positive case for `true` will help the readability.